service/tutorial/order/NewTutorialServices.xml

```diff
@@ -0,0 +1,322 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<services xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+          xsi:noNamespaceSchemaLocation="http://moqui.org/xsd/service-definition-2.1.xsd">
+    
+    <!--    Assignment 2-->
+
+    <!--    Created Order-->
+    <service verb="create" noun="OrderData" displayName="Create an order" type="inline">
+        <in-parameters>
+            <parameter name="orderName" required="true"/>
+            <parameter name="currencyUomId" default-value="USD"/>
+            <parameter name="salesChannelEnumId"/>
+            <parameter name="statusId" default-value="orderPlaced"/>
+            <parameter name="productStoreId"/>
+            <parameter name="placedDate" required="true"/>
+            <parameter name="approvedDate"/>
+        </in-parameters>
+        <out-parameters>
+            <parameter name="orderId"/>
+        </out-parameters>
+        <actions>
+            <entity-find entity-name="mantle.party.Party">
+                <econdition field-name="partyId" value="CustJqp"/>
+            </entity-find>
+            <service-call name="create#mantle.order.OrderHeader" in-map="context" out-map="context"/>
+        </actions>
+    </service>
+
+    <!--    Added Order Items-->
+
+    <service verb="add" noun="orderAdd" displayName="Added an order" type="inline">
+        <in-parameters>
+            <parameter name="orderId" required="true"/>
+            <parameter name="partName"/>
+            <parameter name="facilityId" required="true"/>
+            <parameter name="shipmentMethodEnumId" default-value="ShMthGround"/>
+            <parameter name="customerPartyId"/>
```

Comment:
Please check the required parameters for this service and update accordingly.

Reply:
completed